### ISSUE TITLE
Medicine wheel

### ISSUE THREAD

**Issue by arecvlohe** [nativesintech/indigemoji#5]
Example: https://www.saulttribe.com/history-a-culture/our-culture/99-medicine-wheel

**Comment by arecvlohe** [comment]
I haven't asked but I was curious how we would handle images that are local as opposed to downloading them from a website. I wasn't sure if maybe we should create an s3 bucket or something for _all_ of the images and we can have a bit more control over it, add to it and what not.

**Comment by delesslin** [comment]
Have you looked at IPFS? I've been playing with it for a few projects. I'm not sure how practical it is but I am swooned by the idea of content-referenced (as opposed to locale-referenced) decentralized storage. $0.02

**Comment by RussWilkie** [comment]
Not sure if this avenue is still being explored, but I did create a medicine wheel on illustrator and uploaded it onto media-wiki for scraping. I opted not to use the linked image since media wiki was asking about publishing rights, but did replicate it.

I wasn't sure what dimensions are typically preferred for the project, I went off what I saw in the height dimensions for a couple of the images and set it to 420 x 420px. If you want any changes I can make the appropriate revisions. Otherwise, I'll move forward with testing this out locally and hopefully get PR opened sometime next week. 

https://commons.wikimedia.org/wiki/File:Medicine-Wheel.png
